doc/generate_cet.py

```diff
@@ -15,6 +129,7 @@ def main():
     f.write(".. toctree::\n  :glob:\n  :maxdepth: 1\n\n  *\n\n")
     subsystem_tree = ET.parse("../sal_interfaces/SALSubsystems.xml")
     for subsystem in testutils.subsystems:
+        has_specific = []
```

Comment:
What is has_specifics? The name makes it sound like a list of boolean. A code comment would help. If it is not a list of boolean then please consider a clearer name.

Reply:
Still want a code comment.

Reply:
What does the list contain? If it is a list of specific topic names then please consider renaming it to something like specific_topics.